subsys/net/lib/lwm2m_client_utils/lwm2m/neighbour_cell_info.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <net/lwm2m.h>
+#include <net/lwm2m_client_utils.h>
+#include <lwm2m_engine.h>
+
+#include <modem/modem_info.h>
+#include <modem/lte_lc.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(lwm2m_neighbour_cell, CONFIG_LWM2M_CLIENT_UTILS_LOG_LEVEL);
+#define MAX_INSTANCE_COUNT CONFIG_LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_INSTANCE_COUNT
+
+#define SIGNAL_MEAS_INFO_PHYS_CELL_ID		0
+#define SIGNAL_MEAS_INFO_ECGI			1
+#define SIGNAL_MEAS_INFO_ARFCN_EUTRA		2
+#define SIGNAL_MEAS_INFO_RSRP_RESULT		3
+#define SIGNAL_MEAS_INFO_RSRQ_RESULT		4
+#define SIGNAL_MEAS_INFO_UE_RXTX_TIMEDIFF	5
```

Comment:
I think it'd make sense to declare them in a header (along with object ID) instead of duplicating definitions here.

Reply:
Good notion. Actually now that I think of it, these aren't even used here, only in the object source-file.